selenium_tests/course_info/page_objects/course_info_search_page_object.py

```diff
@@ -81,3 +87,16 @@ def select_course(self, cid=None, title=None):
         # loading the results can take a long time, so explicitly wait longer
         WebDriverWait(self._driver, 30).until(lambda s: s.find_element(
             *CoursePeoplePageLocators.ADD_PEOPLE_BUTTON).is_displayed())
+
+    def get_text(self, course_code_text):
+        """
+        :return: The text in the web element in the course code column
+                 else return None
+        """
+        element = self.find_element(
+            *Locators.COURSE_CODE_TEXT(course_code_text))
+
+        if element.text > 0:
```

Comment:
I think maybe this should be `if len(element.text) > 0`. And do we want to `strip()` the excess spaces first?